app/interfaces/api/v2/mi/agfs_scheme_ten_claims.rb

```diff
@@ -27,12 +27,12 @@ def claims
             desc 'Retrieve claims created with a scheme 10 rep order'
             get do
               if params[:format].eql?('csv')
-                records = claims.count.zero? ? [{ 'No scheme ten claims filed on' => params[:date] }] : claims
+                records = claims.any? ? claims : [{ 'No scheme ten claims filed on' => params[:date] }]
```

Comment:
Just FYI, you can use `.none?` as the exact opposite of `.any?`. Having said that, having the desired result first makes more sense.

Reply:
will leave as `any?` for now - I prefer positive expressions myself